login-workflow/src/components/WorkflowCard/Workflow.test.tsx

```diff
@@ -132,6 +132,12 @@ describe('WorkflowCardInstructions tests', () => {
         render(<WorkflowCardInstructions instructions="Test" divider={true} />);
         expect(screen.getByText('Test')).toBeInTheDocument();
     });
+
+    it('renders with divider', () => {
```

Comment:
We should test the opposite case as well (since divider is true, we should test that it doesn't show when divider is set to false).

Reply:
Added test case.